test/unit/spec/DeviceFingerprint_spec.js

```diff
@@ -42,7 +42,7 @@ function ($, Q, Expect, $sandbox, DeviceFingerprint) {
 
     it('returns a fingerprint if the communication with the iframe is successfull', function (done) {
```

Comment:
this test is failing in Travis, take a look please. It might be that the baseurl is not the same.
If you need more ideas about fingerprint tests, Len just added a new [test spec in auth js](https://github.com/okta/okta-auth-js/blob/master/test/spec/fingerprint.js) that you can look at

Reply:
Fixed